salt/data-pipeline/bigquery-views.sls

```diff
@@ -0,0 +1,39 @@
+clone biquery-views repo:
+    builder.git_latest:
+        - name: git@github.com:elifesciences/data-pipeline-bigquery-views
+        - identity: {{ pillar.elife.projects_builder.key or '' }}
+        - rev: {{ salt['elife.rev']() }}
+        - branch: {{ salt['elife.branch']() }}
+        - target: /opt/data-pipeline-bigquery-views
+        - force_fetch: True
+        - force_checkout: True
+        - force_reset: True
+
+    cmd.run:
+        - cwd: /opt/data-pipeline-bigquery-views
+        - name: docker-compose build
+        - require:
+            - builder: clone biquery-views repo
+        - unless:
+            - test -d /vagrant
+
+    file.directory:
+        - name: /opt/data-pipeline-bigquery-views
+        - user: {{ pillar.elife.deploy_user.username }}
+        - group: {{ pillar.elife.deploy_user.username }}
+        - recurse:
+            - user
+            - group
+        - require:
+            - cmd: clone biquery-views repo
+
+re-materialise views daily:
+    cron.present:
+        - user: {{ pillar.elife.deploy_user.username }}
+        - name: /opt/data-pipeline-bigquery-views/docker-views-cli.sh --dataset={{ pillar.elife.env }} materialize
```

Comment:
actually, don't we need to change to the corresponding directory? alternatively we could modify `docker-views-cli.sh` to change to the directory it thinks it is in.

Reply:
good catch